### ISSUE TITLE
Renaming invalid field names such as 'delegate'

### ISSUE THREAD

**Issue by siketyan** [Cysharp/csbindgen#14]
When we have a struct field with an invalid name in C# world, it causes compilation failure.

For example:

```rs
pub struct NfcCard {
    delegate: extern "C" fn(ByteArray) -> ByteArray,
}
```

This code is converted to:

```cs
[StructLayout(LayoutKind.Sequential)]
internal unsafe partial struct NfcCard
{
    public delegate* unmanaged[Cdecl]<ByteArray, ByteArray> delegate;
}
```

In this context, the name `delegate` is the reserved word in C#, so the compiler says:

```
error CS1519: Invalid token 'delegate' in class, record, struct, or interface member declaration
```

It would be great if csbindgen renames the reserved words of C# I think.

**Comment by neuecc** [comment]
Thanks, I've released 1.4.0.
It generates like

```csharp
[StructLayout(LayoutKind.Sequential)]
internal unsafe partial struct NfcCard
{
    public delegate* unmanaged[Cdecl]<ByteArray, ByteArray> @delegate;
}
```

**Comment by siketyan** [comment]
Thank you for the quick fix!
